verify-service-provider.yml

```diff
@@ -55,7 +58,12 @@ samlPrimaryEncryptionKey: ${SAML_PRIMARY_ENCRYPTION_KEY:-}
 # This only needs to be set if during key rotations (for example if your primary encryption certificate is about to expire)
 samlSecondaryEncryptionKey: ${SAML_SECONDARY_ENCRYPTION_KEY:-}
 
+# Enable European identities (optional)
+# Include this section to enable your VSP to process European identities.
 europeanIdentity:
+  # `enabled: true` to allow your VSP to process European identities
+  # `enabled: false` means your VSP won't process European identities, but
+  # the section must contain valid key-value pairs.
   enabled: ${EUROPEAN_IDENTITY_ENABLED}
   hubConnectorEntityId: ${HUB_CONNECTOR_ENTITY_ID}
```

Comment:
Most of this can be configured with the Hub Environment now.

Reply:
That's right. The section now only requires the `enabled` flag. The rest is now automagically configured (but it can still be overwritten)

Reply:
Cool! Leaving only the `enabled` flag 👍